src/pomodoro/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+type ButtonProps = {
+  className: 'Button Button--type-square' | 'Button Button--type-control';
```

Comment:
If this is supposed to be BEM it would probably need to be something like. `button button--square` or `button button--control`. You should avoid any letter casing in css classes

Reply:
I saw this in React docs, so I used to think, that's the only valid way. https://reactjs.org/docs/composition-vs-inheritance.html